src/editors/containers/EditorContainer/index.jsx

```diff
@@ -36,7 +36,12 @@ export const EditorContainer = ({
         confirmAction={(
           <Button
             variant="primary"
-            onClick={handleCancel}
+            onClick={() => {
+              handleCancel();
+              if (returnFunction) {
```

Comment:
The confirmation modal is forced close when the editor is used for an editor xblock because the component in no longer rendered. Therefore, we only need the modal to be closed when the editors are being used outside of the traditional xblock editors. When the confirmation is closed in the traditional modal, it closes before the redirect for the page occurs. This behavior is confusing to the user.